ros2-release/ros2-release.spec

```
@@ -0,0 +1,37 @@
+Name:           ros2-release
+Version:        1.0.0
+Summary:        Packages for ROS 2 main repository configuration
+Release: 1%{?release_suffix}
+BuildArch: noarch
+License: ASL 2.0
+URL: https://github.com/ros-infrastructure/ros-apt-source
+Source0: README.md
+Requires:  epel-release
+
+Source10:       RPM-GPG-KEY-ROS2
+Source20:       ros2.repo
+Source21:       ros2-testing.repo
+
+
+
+ 
+%description
+This package contains the ROS 2 repository configuration and GPG key.
+ 
+%prep
+%setup -q -c -T
+cp -a %{SOURCE0} .
+ 
```

It isn't an error, but it is an rpmlint warning to omit the `%build` section entirely.

```suggestion
 
 %build
 
```